modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/enums/AndesConfiguration.java

```diff
@@ -546,24 +546,42 @@
     PERFORMANCE_TUNING_FAILOVER_VHOST_SYNC_TASK_INTERVAL("recovery" +
             "/vHostSyncTaskInterval", "900", Integer.class),
 
+    /**
+     * Enable/ Disable expiry check in the DLC
+     */
+    PERFORMANCE_TUNING_ALLOW_EXPIRATION_CHECK_IN_DLC
+            ("performanceTuning/messageExpiration/allowExpiryCheckInDLC", "false", Boolean.class),
```

Comment:
name of the configuration setting doesn't make sense when a user see it (unless he/she goes and read about it in the documentation). We need a better name.